src/rpc/cli.cpp

```diff
@@ -250,6 +249,12 @@ void cli::getline( const fc::string& prompt, fc::string& line)
             FC_THROW_EXCEPTION( fc::eof_exception, "" );
          line = line_read;
          // we don't need here to add line in editline's history, cause it will be doubled
+         if (cli_check_secret(line_read)) {
+            el_no_echo = 1;
+            line_read = readline("Enter password: ");
```

Comment:
Must `free(line_read)` before re-using it

Reply:
done